### PR TITLE
repair SCE ledger updates

### PR DESCRIPTION
**Pull request by damip**

### CODE REVIEW

File: massa-execution/src/sce_ledger.rs [hunk 1]
```diff
@@ -32,7 +32,9 @@ impl SCELedgerEntry {
         }
 
         // module
-        self.opt_module = update.update_opt_module.clone();
+        if let Some(opt_module) = &update.update_opt_module {
+            self.opt_module = opt_module.clone();
+        }
```
Comment (adrien-zinger) [inline massa-execution/src/sce_ledger.rs:37]: Can just be 
```rust
        if let Some(module) = update.update_opt_module {
            self.opt_module = Some(module);
        }
```

without manage all the process an Option right? You never make a real use of the super Option

**Comment by damip** [comment]
The compiler in the CI crashes  :/

**Comment by adrien-zinger** [comment]
> The compiler in the CI crashes :/

Do you think that we should define a version of rust for this episode?

**Review by adrien-zinger — APPROVED** [review]

**Comment by damip** [comment]
> > The compiler in the CI crashes :/
> 
> Do you think that we should define a version of rust for this episode?

good idea. Can you make a PR ?

**Comment by AureliaDolo** [comment]
> > > The compiler in the CI crashes :/
> > 
> > 
> > Do you think that we should define a version of rust for this episode?
> 
> good idea. Can you make a PR ?

Isn't it a risk that we forget to update it ?

**Comment by AureliaDolo** [comment]
bors merge

**Comment by damip** [comment]
> > > > The compiler in the CI crashes :/
> > > 
> > > 
> > > Do you think that we should define a version of rust for this episode?
> > 
> > 
> > good idea. Can you make a PR ?
> 
> Isn't it a risk that we forget to update it ?

yes, that's why we need to make an issue to roll it back

**Pull request merged**